styles/dowhop-selector-style.css

```diff
@@ -13,14 +13,17 @@
 */
 
 .Button-Style {
+
   background-color: red;
-  left: 66px;
+  left: 10px;
+  bottom: 20px;
   position: absolute;
   height: 40px;
   min-height: 19px;
   width: 142px;
   color: white;
   font-size: 18px;
+  padding-top: 10px;
 }
 
```

Comment:
Nice work on the styles. 👍  
Some position issues in my browser are caused by the left, bottom, position, and padding-top properties. When I deleted them the text centers and the button positions back into place. 
Can we add a pseudo class for :focus this will help with accessibility. 
We need to double check with design if the button color is crimson #ec1928 or red #ff0000. My color picker keeps changing its mind. 
The font is rendering as Roboto by the MDL stylesheet. It should be:
font-family: 'Avenir Light', Avenir Light, Arial, sans-serif;